docs/reference/commandline/service_inspect.md

```diff
@@ -0,0 +1,145 @@
+<!--[metadata]>
++++
+title = "service inspect"
+description = "The service inspect command description and usage"
+keywords = ["service, inspect"]
+[menu.main]
+parent = "smn_cli"
++++
+<![end-metadata]-->
+
+**Warning:** this command is part of the Swarm management feature introduced in Docker 1.12, and might be subject to non backward-compatible changes.
+
+# service inspect
+
+```Markdown
+Usage:	docker service inspect [OPTIONS] SERVICE [SERVICE...]
+
+Inspect a service
+
+Options:
+  -f, --format string   Format the output using the given go template
+      --help            Print usage
+  -p, --pretty          Print the information in a human friendly format.
+```
+
+
+Inspects the specified service. This command has to be run targeting a Manager
```

Comment:
thanks, copied that from the original content, so thought it was meant to be like this; I'll update